zipline/sources/benchmark_source.py

```diff
@@ -77,27 +79,27 @@ def _validate_benchmark(self, benchmark_asset):
         # error suggesting that the user pick a different asset to use
         # as benchmark.
         stock_dividends = \
-            self.data_portal.get_stock_dividends(self.benchmark_sid,
+            self.data_portal.get_stock_dividends(self.benchmark_asset,
```

Comment:
`get_stock_dividends` accepts either `Asset` or an `int`. (It calls `int` on the `sid` value before creating the sql query.)
The docstring should probably be updated on that method to reflect that either is accepted; though in general we have aimed to pass `Asset`s to `DataPortal` methods.